docs/src/pages/components/typography.mdx

```diff
@@ -1 +1,186 @@
+import { Link } from 'gatsby';
+
+<br />
+
 # Typography
+
+The `Typography` component lets you easily add unified text styles to the page without having to create
+a lot of extra css.
+
+Using `Typography` without modifying any of its props will wrap any text element in a `span` tag.
+These are the default styles for Typography:
+
+* `font-family: Avenir Next, Segoe UI, Helvetica Neue, Helvetica, Roboto,sans-serif;`
+* `font-size: 1rem;`
+* `line-height: 1.5rem;`
+* `text-align: inherit;`
+* `text-transform: none;`
+* `color: inherit;`
+
+<br />
+
+#### Inline Code Editor
```

Comment:
We should add a descriptor in here about the reason we have this notion of `scale`. It's (1) our way of mitigating the fact that semantic HTML tagging doesn't always get the same text styles and we needed a mechanism for handling that and (2) we needed a consistent way to render common fonts regardless of their enclosing tags for CTAs.

Reply:
Updated description: "This value directly affects the size and lineHeight props directly, ensuring
                that there is no overlap of text between lines no matter what value is input.
                Often, semantic HTML tagging doesn't always get the same text styles and this helps
                alleviate that problem. It also assists in rendering common fonts regardless of 
                their enclosing tags."

Reply:
I've added a paragraph to this effect.